windows/vhba_hot_add_remove/paravirtual_vhba_device_ops.yml

```diff
@@ -13,18 +13,16 @@
     - "{{ testing_vars_file | default('../../vars/test.yml') }}"
   tasks:
     - block:
-        - name: "Set fact of skip test to True when no VMware tools installed"
-          ansible.builtin.set_fact:
-            skip_pvscsi_no_vmtools: true
-        - name: "Set fact of skip test to False when there is pvscsi inbox driver"
-          ansible.builtin.set_fact:
-            skip_pvscsi_no_vmtools: false
-          when:
-            - guest_os_with_inbox_drivers
-
         - include_tasks: ../setup/test_setup.yml
+
+        # Skip test case if no pvscsi driver in guest OS, no matter it's from inbox or tools
+        - include_tasks: ../../common/skip_test_case.yml
           vars:
-            skip_test_no_vmtools: "{{ skip_pvscsi_no_vmtools }}"
+            skip_msg: "Test case '{{ current_testcase_name }}' is blocked because guest OS has no pvscsi driver neither inbox nor VMware tools"
```

Comment:
because guest OS has no pvscsi driver neither inbox nor VMware tools -> because guest OS has no inbox PVSCSI driver or VMware tools is not installed.